rust/src/ledger/diff/account.rs

```diff
@@ -216,9 +216,9 @@ impl AccountDiff {
                                     ) => {
                                         if let Some(fee) = cb.t.fee_transfer_receiver_balance {
                                             let fee_transfer_receiver_balance = fee.t.t.t;
-                                            if (*total_fee)
-                                                .saturating_sub(MAINNET_ACCOUNT_CREATION_FEE.0)
-                                                == fee_transfer_receiver_balance
+                                            if *total_fee >= MAINNET_ACCOUNT_CREATION_FEE.0
```

Comment:
This implies that one cannot transfer less than the account creation fee. However, that is only true when the account has not yet been created. Is that true, here?